agent/api/eni.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package api
+
+import (
+	"github.com/aws/amazon-ecs-agent/agent/acs/model/ecsacs"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+type ENIAttachment struct {
+	AttachmentArn    *string `json:"attachmentArn"`
+	AttachStatusSent bool    `json:"attachSent"`
+	MacAddress       *string `json:"macAddress"`
+}
+
+type ENI struct {
+	ID            string `json:"ec2Id"`
+	IPV4Addresses []*ENIIPV4Address
```

Comment:
The name here and above are all extracted from the `acs/model` to keep consistency, I think it is reasonable to keep the same name in the `api`package.